lib/chef/application/apply.rb

```diff
@@ -137,11 +137,11 @@ def parse_json
 
   def read_recipe_file(file_name)
     if file_name.nil?
-      Chef::Application.fatal!("No recipe file was provided", 1)
+      Chef::Application.fatal!("No recipe file was provided")
```

Comment:
should you be specifying ` Chef::Exceptions::RecipeNotFound.new` as yo do below?